Firestore/Example/Tests/Model/FSTMutationTests.mm

```diff
@@ -69,9 +71,8 @@ - (void)testDeletesValuesFromTheFieldMask {
   FSTDocument *baseDoc = FSTTestDoc("collection/key", 0, docData, NO);
 
   DocumentKey key = testutil::Key("collection/key");
-  FSTFieldMask *mask = [[FSTFieldMask alloc] initWithFields:{testutil::Field("foo.bar")}];
   FSTMutation *patch = [[FSTPatchMutation alloc] initWithKey:key
-                                                   fieldMask:mask
+                                                   fieldMask:FieldMask({testutil::Field("foo.bar")})
```

Comment:
Nit: just `FieldMask{testutil::Field("foo.bar")}` should do.

Reply:
done